services/gundeck/test/integration/API.hs

```diff
@@ -71,54 +71,54 @@ tests s =
     "API tests"
     [ testGroup
         "Push"
-        [ test s "Register a user" $ addUser,
-          test s "Delete a user" $ removeUser,
-          test s "Replace presence" $ replacePresence,
-          test s "Remove stale presence" $ removeStalePresence,
-          test s "Single user push" $ singleUserPush,
+        [ test s "Register a user" addUser,
+          test s "Delete a user" removeUser,
+          test s "Replace presence" replacePresence,
+          test s "Remove stale presence" removeStalePresence,
+          test s "Single user push" singleUserPush,
```

Comment:
I don't like this change, because I'm obsessed with symmetry, but I like HLint too much to object.  :)